std/format/internal/read.d

```diff
@@ -83,7 +83,6 @@ if (isInputRange!Range && is(T == typeof(null)))
     return parse!T(input);
 }
 
-/// ditto
```

Comment:
Why remove these?

Reply:
Public comment for a non public function. And it does not refer to anything (ditto what?). In my opinion it's just a leftover from former days.